frontend/src/components/Comments/CommentView.js

```diff
@@ -0,0 +1,32 @@
+import {useSelector} from 'react-redux';
+import CommentInput from "./CommentInput"
+import "../../style/Comments.css";
+import {Avatar} from "antd";
+import {UserOutlined} from "@ant-design/icons";
+
+function CommentView() {
+    const comments = useSelector((state) => state.comments);
+
+    return (
+        <div className="comment-main">
+            <h3>Comments</h3>
+            <div className="comment-add">
+                <div><CommentInput/></div>
+            </div>
+            <div className="comment-list">
+                {comments.map((comment) => (
+                    <div className="comment-container" >
```

Comment:
how about extracting this to be a component?

Reply:
I'm sorry, but I'm not sure what exactly needs to be a component here, and why? Are you referring to everything inside and include the comment-list div?

Reply:
Because I feel like they are closedly knitted and are subject to future change?